tests/Resources/Entity/AllAvailableActionsAnnotatedModel.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Opstalent\CrudBundle\Tests\Resources\Entity;
+
+use Opstalent\CrudBundle\Annotation\Entity;
+use Opstalent\CrudBundle\Annotation\Field;
+
+/**
+ * Class AllAvailableActionsAnnotatedModel
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ * @Entity(actions={"addable", "deletable", "editable", "getable", "listable"})
+ */
+class AllAvailableActionsAnnotatedModel
```

Comment:
not needed if we're not testing integrations

Reply:
fixed